.github/workflows/sourcegraph.yml

```diff
@@ -18,11 +18,25 @@ jobs:
           java-version: 8
           cache: 'sbt'
 
+      - name: Publish CLI locally
+        run: sbt publishLocal dumpScipJavaVersion
+
+      - name: Produce SCIP data of scip-java repository
+        run: sbt -Dscip-java-version=$(cat VERSION) sourcegraphCompile
```

Comment:
An alternative to `-Dscip-java-version` is to `set every sourcegraphScipJavaVersion := version.value` inside a command. We can add a command in the root of build.sbt that does something like this

```
commands +=
  Command.command("indexMinimized") { s =>
    "cli/publishLocal" ::
      "set every sourcegraphScipJavaVersion := version.value" ::
      "sourcegraphCompile" :: s
  }
```

I haven't tested this and we can do it separately from this PR. It would be convenient to have a command in the sbt shell to generate an index for local debugging.

Reply:
Yeah, I realised the same later.

Once we confirm that correct scip is being uploaded, I'll go back and make this nicer.